test/common/pixel-tests

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -eux
+
+SUBDIR=test/reference
+
+GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
+GITHUB_REPOSITORY="${GITHUB_BASE%/*}/pixel-test-reference"
+CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
+PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
+
+message() {
+    [ "${V-}" != 0 ] || printf "  %-8s %s\n" "$1" "$2"
+}
+
+get_index_gitlink() {
+    if ! git ls-files -s "$1" | egrep -o '\<[[:xdigit:]]{40}\>'; then
+        echo "*** couldn't read gitlink for file $1 from the index" >&2
+        exit 1
+    fi
+}
+
+subgit() {
+    git -C "$SUBDIR" "$@"
+}
+
+setup() {
+    # Cases:
+    #
+    # - $SUBDIR doesn't exist in index => fail
+    # - $SUBDIR is no gitlink in index => fail
```

Comment:
The code here in setup() does not even look at git, so these two comments should move to the place where the index is checked.